docs/reference/migration/migrate_7_0/indices.asciidoc

```diff
@@ -78,3 +78,12 @@ The parent circuit breaker defines a new setting `indices.breaker.total.use_real
 heap memory instead of only considering the reserved memory by child circuit breakers. When this
 setting is `true`, the default parent breaker limit also changes from 70% to 95% of the JVM heap size.
 The previous behavior can be restored by setting `indices.breaker.total.use_real_memory` to `false`.
+
+==== `fix` value for `index.shard.check_on_startup` is removed
+
+`elasticsearch-shard remove-corrupted-segments` tool has to be used instead of
+`index.shard.check_on_startup: fix` setting.
```

Comment:
Let's consistently refer to it as the "elasticsearch-shard" tool. I'd just say:

Use the `elasticsearch-shard` tool to remove corrupted Lucene index segments.